### DISCUSSION TITLE
float type support prisma 1

### DISCUSSION THREAD

**Discussion by byteab** [prisma/prisma#2813] [Q&A] [answered]
I have created a mongoose float type field but now when I query the document with prisma v1 it logs this error.
`Not implemented: com.prisma.shared.models.TypeIdentifier$Float$@6d74c5c7BsonInt32{value=4440}`
how can I tackle this issue. what type should I use to represent float number that not interfere with prisma

**Comment by janpio** [comment, marked as answer]
This is the GitHub repository for Prisma 2. For Prisma 1 questions you should probably ask at https://github.com/prisma/prisma1

**Reply by byteab** [reply]
@ok

**Comment by ludralph** [comment]
Hi @byteab,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
